chiasim/validation/chainview.py

```diff
@@ -141,6 +143,7 @@ def for_genesis_hash(cls, genesis_hash: HeaderHash, unspent_db: UnspentDB):
 
     try:
         newly_created_block_index = chain_view.tip_index + 1
+        dob = math.floor(time.time() * 1000)
```

Comment:
We need to factor out this global and assume this `dob` value is being passed into via the context instead.

Reply:
done